torch_tvm/compiler.cpp

```diff
@@ -419,14 +450,49 @@ void TVMCompiler::run(Stack& stack) {
     // This help performance.
     build_config->partition_const_loop = true;
     build_f(tvm_func, target_map, tvm::Target::Create(host_));
-    std::string json = json_f();
     tvm::runtime::Module mod = mod_f();
+    std::string json = json_f();
+    if (debug_) {
```

Comment:
Maybe we should just use a class, say DebugLogger to isolate these code.

Reply:
Let me think.